.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,18 @@
+name: helm-release
+
+on:
+  push:
+    branches:
+      - helm-release-test
```

Comment:
Why only on this specific branch? What's the expected workflow?

Reply:
it was for testing only, not going to enable on main until I'm 100% sure it doesn't break our CI/CD